game.py

```diff
@@ -38,23 +105,50 @@ def _made_accusation(self, player, suggestion):
 
 
 class Card(object):
-	pass
 	'''
-	type
-	name
-	?owner
+	VARIABLE CLASS:
+	type = <Room, Suspect, Weapon>
+	name = <Miss Scarlett, Lead Pipe, Study .. >
+	owner - player that holds the card
+	dom - dom is list of names card can take on or 
+		- only contains name of card
+	curdom - current domain
+	assignedType - Type of the card is known
 	'''
-	def __init__(self, type, name):
+	def __init__(self, type=None, name=None, domain=[]):
 		'''
-		init it with the things
+		Initialize a card with a type = <Room, Suspect, Weapon>
+		and a name = <Miss Scarlett, Lead Pipe, Lounge, ...>
 		'''
-		pass
+		self.type = type
+		self.name = name
+		self.player = None
+
+		self.dom = list(domain)
```

Comment:
When do you use domain, as well as assignedType? Not sure about the implementation of these here